app/helpers/refusal_advice_helper.rb

```diff
@@ -4,4 +4,20 @@ def refusal_advice_actionable?(action, info_request:)
     return true unless action.target.key?(:internal)
     current_user && current_user == info_request&.user
   end
+
+  def refusal_advice_form_data(info_request)
+    return {} unless info_request
+
+    { refusals: latest_refusals_as_json(info_request) }
+  end
+
+  private
+
+  def latest_refusals_as_json(info_request)
```

Comment:
We're not actually converting to JSON here. Given the other comment below, I _think_ we can condense the `private` methods here down to:

```ruby
def latest_refusals(info_request)
  Array(info_request.refusals.reject(&:empty?).last).map(&:to_param)
end
```